update_version.yml

```
@@ -0,0 +1,29 @@
+name: Update Version on Docker Files
+
+env:
+  PRJ_VER: ${{ github.event.inputs.project_version }}
+
+
+
+on:
+ workflow_dispatch:
+    inputs:
+      project_version:
+        description: 'Version number to be released e.g 10.0.3'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: sudo apt install libcurl4-openssl-dev libssl-dev
+
+      - name: Clone Tools branch
+        run: git clone --branch tools-transition-update-docker https://github.com/citusdata/tools.git
```

```suggestion
      - name: Install Tools
        run: |
        	git clone --branch tools-transition-update-docker https://github.com/citusdata/tools.git
        	make -C tools install
```

You also need to install the scripts I think

---

In this case there is no need to install scripts but I will add the package_automation directory into PYTHONPATH library